### ISSUE TITLE
Update PHP deps with fixed Critical Vulns

### ISSUE THREAD

**Issue by yuriy-yarosh** [goryn-clade/pathfinder#180]
This pathfinder fork has some pretty old deps worth updating

![grype output](https://media.discordapp.net/attachments/900844184028725299/1123643141413679196/image.png)

Can we do something about it ?

**Comment by TyrHeimdalEVE** [comment]
Yes, however it's not very easy to solve as code needs a lot of restructuring to behave with PHP 8+.

I've tried numerous solutions to get us there (or at least as close as possible), and with Alpine no longer publishing PHP 7.x in their repositories that puts us in a rough spot.

One possible direction would be to switch to for example Ubuntu 20.04 LTS which will provide us with backported security updates until 2025-04-01. But it requires some rework of the structuring, as service related paths are different between the distros.

I do however have this on my scope, and have already re-worked substantial parts which addresses majority of the above and will prioritize for the next release which I _hope_ we can release within a few weeks.
